runner/src/main.rs

```diff
@@ -310,7 +310,7 @@ fn build_wasm_module(name: &str, target: &Target, example_name: &str) -> Step {
 
 fn build_server(opt: &BuildServer) -> Step {
     match opt.server_variant.as_str() {
-        "base" | "logless" => Step::Multiple {
+        "base" | "logless" | "no-introspection-server" => Step::Multiple {
```

Comment:
Currently this would strip out the dynamic introspection client (eg everything behind `/dynamic` route), but would still include the introspection _server_ itself. 

If we would like to keep that as is we should probably rename this. 

I'm not entirely sure what's the usecase for this flag is though? It cuts out some but not all introspection functionality.

Reply:
Ah yes, my bad, the name should probably be "no-introspection-client" to match the `oak_introspection_client` feature that it's turning off.

The primary use case is to allow local building and testing without requiring npm to be installed locally.

Reply:
Fair. :) Though I would caution that you already need Node.js/NPM to run our formatting scripts (prettier)